src/frontend/components/Chat/SharedChatComponents/InputDisplayNameOverlay/index.tsx

```diff
@@ -57,16 +58,15 @@ const messages = defineMessages({
 });
 
 interface InputDisplayNameOverlayProps {
-  setInputBarActive: React.Dispatch<React.SetStateAction<boolean>>;
   setOverlay: React.Dispatch<React.SetStateAction<boolean>>;
 }
 
 export const InputDisplayNameOverlay = ({
-  setInputBarActive,
   setOverlay,
 }: InputDisplayNameOverlayProps) => {
   const intl = useIntl();
   const [alertsState, setAlertsState] = useState<string[]>([]);
+  const { setDisplayName } = useChatItemState.getState();
```

Comment:
This is not how a zustand store should be used in a component. Refer to the documentation to know how: https://github.com/pmndrs/zustand#selecting-multiple-state-slices

Reply:
yes thanks you, I changed it